openstack/assets/dashboards/openstack-controller.json

```
@@ -0,0 +1,693 @@
+                        "type": "area"
+                    },
+                    {
+                        "q": "avg:openstack.nova.limits.max_total_cores{$project}",
+                        "style": {
+                            "palette": "warm",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "time": {
+                "live_span": "1h"
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Cores used vs max by $project",
+            "type": "timeseries",
+            "width": 46,
+            "x": 69,
+            "y": 74
+        },
+        {
+            "height": 14,
+            "id": 9,
+            "legend": true,
+            "legend_size": "0",
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:openstack.nova.limits.total_instances_used{$project}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "area"
+                    },
+                    {
+                        "q": "avg:openstack.nova.limits.max_total_instances{$project}",
+                        "style": {
+                            "palette": "warm",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "time": {
+                "live_span": "1h"
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Instances used vs max by $project",
+            "type": "timeseries",
+            "width": 46,
+            "x": 69,
+            "y": 92
+        },
+        {
+            "height": 14,
+            "id": 10,
+            "legend": true,
+            "legend_size": "0",
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:openstack.nova.limits.total_ram_used{$project}*1024000",
```

```suggestion
                        "q": "avg:openstack.nova.limits.total_ram_used{$project}",
```
Don't multiply the value, the backend should know that this value is in GiB and will convert and display the correct units on the graph.

---

@FlorianVeaux this is identical to the way this widget is displayed for the default OpenStack dashboard currently in production.

---

Just noticed that this differs from the regular OpenStack one. Committed changes.